modules/http-helper/http_helper.go

```diff
@@ -26,6 +27,8 @@ func HttpGet(t *testing.T, url string) (int, string) {
 func HttpGetE(t *testing.T, url string) (int, string, error) {
 	logger.Logf(t, "Making an HTTP GET call to URL %s", url)
 
+	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
We actually have a few tests in our IaC library that checks the TLS cert is valid, so we definitely don't want to do this globally. I think it would be more flexible to accept the TLS config as a parameter to the function. This allows you to not only optionally disable TLS verification, but also support custom CAs.

This will be a backwards incompatible change, but I think we can live with it given the additional functionality it enables.

Note that this requires updating all the functions in this file to support accepting the parameter.

Reply:
I just added your suggested fixes in #336 @yorinasub17 

This is blocking some of our testing as well, so I thought I would pick it up and finish it.